cmd/helm/main.go

```diff
@@ -205,35 +203,22 @@ func main() {
 		os.Exit(1)
 	}
 
-	commonBundleProvisionerOptions := []bundle.Option{
-		bundle.WithUnpacker(unpacker),
-		bundle.WithFinalizers(bundleFinalizers),
-		bundle.WithStorage(bundleStorage),
-	}
-
 	cfgGetter := helmclient.NewActionConfigGetter(mgr.GetConfig(), mgr.GetRESTMapper(), mgr.GetLogger())
 	acg := helmclient.NewActionClientGetter(cfgGetter)
 	commonBDProvisionerOptions := []bundledeployment.Option{
 		bundledeployment.WithReleaseNamespace(systemNamespace),
+		bundledeployment.WithFinalizers(bundleFinalizers),
 		bundledeployment.WithActionClientGetter(acg),
 		bundledeployment.WithStorage(bundleStorage),
 	}
 
-	if err := bundle.SetupWithManager(mgr, systemNsCluster.GetCache(), systemNamespace, append(
-		commonBundleProvisionerOptions,
-		bundle.WithProvisionerID(helm.ProvisionerID),
-		bundle.WithHandler(bundle.HandlerFunc(helm.HandleBundle)),
-	)...); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", rukpakv1alpha1.BundleKind, "provisionerID", helm.ProvisionerID)
-		os.Exit(1)
-	}
-
-	if err := bundledeployment.SetupWithManager(mgr, append(
+	if err := bundledeployment.SetupWithManager(mgr, systemNsCluster.GetCache(), systemNamespace, append(
 		commonBDProvisionerOptions,
 		bundledeployment.WithProvisionerID(helm.ProvisionerID),
+		bundledeployment.WithUnpacker(unpacker),
```

Comment:
Same question here that I asked in `core`: put this in the common options slice?